app/views/posts/index.html.erb

```diff
@@ -1,7 +1,105 @@
-<h1>All Posts</h1>
-<p><%= @placeholder_text %></p>
-<ul>
-  <% @posts.each do |post| %>
-    <li><%= link_to post.title, post_path(post) %></li>
-  <% end %>
-</ul>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>All Posts</title>
+  <style>
+    * {
+      box-sizing: border-box;
+      padding: 0;
+      margin: 0;
+      font-family: sans-serif;
+    }
+
+    .posts-list {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      width: 100%;
+      margin: 30px auto;

```

Comment:
- [ ] Please kindly remove this CSS code that you newly added to your work after the last review, this will make your codebase to be neat. 👍 Placing CSS in a separate file allows developers to manage styles more efficiently.